config/initializers/time_zone.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+# Raise if invalid zone is specified
+Time.find_zone!(ENV.fetch('DEFAULT_TIME_ZONE', nil))
```

Comment:
Future refactor here -- we have a series of open PRs to move app wide config like this into yml files that are loaded/validated once at app launch time (via `config_for`) ... if/when those are merged, this is a good setting to go into something like that as well.